src/features/sendEmail/SendEmail.tsx

```diff
@@ -31,9 +38,36 @@ export default function SendMessageTo() {
     setCharactersRemaining(500 - inputValue.length);
   };
 
+  const sendEmailHandle = () => {
+    if (!address) return;
+    sendEmail({
+      emailSubject: messageObject,
+      emailContent: value,
+      protectedData: address,
```

Comment:
:bug: the address used here is the user address, it must be a protected data address